rector.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\ClassConstFetch\RenameClassConstFetchRector;
+use Rector\Renaming\ValueObject\RenameClassConstFetch;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/examples',
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/tools',
+        __FILE__,
+    ]);
+
+    // ReadPreference::RP_* constants are deprecated in favor of the string constants
```

Comment:
Consider adding a reference to https://jira.mongodb.org/browse/PHPC-1489 and optionally https://jira.mongodb.org/browse/PHPC-1021.